joomla/joomla-cms
# [4.0] Fix category edit page title when Associations toolbar fails

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/27025 and https://github.com/joomla/joomla-cms/pull/27029#issuecomment-552076368

### Summary of Changes
Correcting redirect links for category edit.


### Testing Instructions
Create a multilingual site
Go to Content > Categories (same for all core components using categories)
Edit `Uncategorised` (Make sure its language is set to `All`)
Click the `Associations` toolbar button


### Before patch

![uncategorised](https://user-images.githubusercontent.com/368084/68498273-514b3300-020b-11ea-99b3-c473cf0a886d.jpg)



### After patch
The correct string is used.

![cattitle](https://user-images.githubusercontent.com/869724/68525602-aeaab700-02d3-11ea-87f8-5b747a0b4c59.gif)
